create_package.py

```diff
@@ -20,19 +20,21 @@
 Package contains server side files directly,
```

Comment:
How much of the changes were "must have" and "I want them"?

Reply:
Not sure how to answer this. This file produces a lot of linter errors for me. So I can either put it to excluded files - that seems to me wrong - or I can fix the linter issues which is what I did. So I'll cowardly pass the decision to @philippe-ynput 🤠